MANIFEST.in

```diff
@@ -6,7 +6,7 @@ graft icons
 graft scripts/pylint_checkers
 graft doc/img
 graft misc
-include qutebrowser/test/testfile
+include qutebrowser/utils/testfile

```

Comment:
The corresponding test is testing [qutebrowser.utils.utils.read_file](https://github.com/The-Compiler/qutebrowser/blob/master/qutebrowser/utils/utils.py#L68-L92), i.e. a function which loads a resource file from the qutebrowser package (or a subpackage) - so that'd kinda defeat the purpose in this case :wink: